cogs/learninggroups.py

```diff
@@ -547,7 +547,7 @@ async def cmd_add_header(self, ctx, arg_course, *arg_name):
             "semester": ("Das Semester, für welches diese Lerngruppe erstellt werden soll."
                          "sose oder wise gefolgt von der zweistelligen Jahreszahl (z. B. sose22)."),
             "status": "Gibt an ob die Lerngruppe für weitere Lernwillige geöffnet ist (open) oder nicht (closed).",
-            "@usermention": "Die so erwähnte Benutzerin wird als Besitzerin für die Lerngruppe gesetzt."
+            "@usermention": "Die so erwähnte Benutzerin wird als Organisatorin der Lerngruppe eingesetzt."

```

Comment:
Ich bin sowohl mit gesetzt als auch mit eingesetzt nicht 100% sicher. 
Wäre da "definiert" vielleicht geeigneter?

Reply:
Meine Vermutung ist, daß das 1-zu-1 aus dem Englischen übersetzt wurde. In der Datei sind noch einige Sätze, die sich mehr Englisch anhören als Deutsch. 
Vielleicht kann man für die sprachliche Überarbeitung ein Issue öffnen? Oder den Scope von diesen Issue vergrößern.

Reply:
Oh ja das klingt einleuchtend. 
Separates allgemeines Sprachissue könnte ich mir vorstellen, würde die Gelegenheit, ein paar Kleinigkeiten hier zu verbessern, aber schon ergreifen.

Reply:
Festgelegt klingt gut!